setup_venv.ps1

```diff
@@ -68,20 +68,23 @@ else
 if ($PyVer -like "*venv*")
 {
   deactivate # make sure we don't update the wrong venv
+  $ErrorActionPreference = 'SilentlyContinue'
   $PyVer = py --list # update list
 }
 
 Write-Host "Python versions found are"
 Write-Host ($PyVer | Out-String) # formatted output with line breaks
-if (!($PyVer -like "*3.11*")) # if 3.11 is not in list
+if (!($PyVer.length -ne 0)) {$p} # return Python --version String if py.exe is unavailable
```

Comment:
the py --list stuff is only needed if people install multiple versions and 3.11 is not the default.
I don't think that this will be the case if you just install from store anyways.
We could handle that stuff more efficiently later, but it will work right now and prefer py.exe